src/main/java/core/basesyntax/activity/ActivityParserImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.activity;
+
+public class ActivityParserImpl implements ActivityParser {
+    @Override
+    public Activities parseActivity(String[] line) {
```

Comment:
It's better to take String as an input. In this case it'll be more generic. Otherways if you receive data from another source you would need to create line just to use this method. Plus empty arr will generate Index out of bound